src/Features/Core/Portable/CodeRefactorings/SyncNamespace/IChangeNamespaceService.cs

```diff
@@ -51,5 +53,10 @@ internal interface IChangeNamespaceService : ILanguageService
         /// a no-op and original solution will be returned.
         /// </remarks>
         Task<Solution> ChangeNamespaceAsync(Document document, SyntaxNode container, string targetNamespace, CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Gets the target namespace for a given document by using the path and any root namespace 
```

Comment:
1. can you give an example?
2. can you doc when you would get null back and waht that means?